app/src/androidTest/java/io/reark/rxgithubapp/activities/utils/SystemAnimations.java

```diff
@@ -1,9 +1,5 @@
 package io.reark.rxgithubapp.activities.utils;
```

Comment:
It was on the old Espresso site, the current one doesn't have it: https://google.github.io/android-testing-support-library/docs/espresso/setup/

It was google's code